### ISSUE TITLE
Placeholder, role of the date field is announced by JAWS as label for DatePicker button if both the Date field and DatePicker are present inside a table. 

### ISSUE THREAD

**Issue by SwatiSaini15** [FreedomScientific/standards-support#840]
### Summary

There is a table where date field and DatePicker both are present inside 1 cell of the table(i.e. 1st column 2nd row).  Date field has been defined using Input type=text, whereas DatePicker has been implemented similar to W3C example. Issue observed while using tab navigation is Placeholder(MM/DD/YYYY) & role(Edit) of the date field is being announced along with DatePicker label for DatePicker button . Whereas with NVDA only the DatePicker label is being announced.
> 
> Example:

<input  type="text" maxlength="10"  placeholder="(mm/dd/yyyy)" autocomplete="off"  tabindex="0" aria-describedby="DatePlaceholder">

<button type="button" class="icon" aria-label=" choose Payroll Date 1 " tabindex="0">

![image](https://github.com/FreedomScientific/standards-support/assets/170329419/6997d305-4e24-41b2-aac6-0c03bd38cf73)

![image](https://github.com/FreedomScientific/standards-support/assets/170329419/2df07578-6391-4881-8159-e3f93d2fe116)

### Expected result
> JAWS should announce DatePicker button as " Choose Payroll Date 1 button"

### Actual result

> JAWS is announcing the DatePicker button as "MM/DD/YYYY Edit Choose Payroll Date 1 button"

### Additional Information

Issue is only bbserved is the Input date field and DatePicker and present inside the table.

#### JAWS version and build number

2023.2302.15
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 124.0.6367.209

**Comment by stevefaulkner** [comment]
@sahilempire please provide a working test case and test it in the latest version of JAWS. If the issue still occurs, please comment here.

**Comment by stevefaulkner** [comment]
closing as no test case provided
